sdk/servicebus/azure-servicebus/azure/servicebus/common/mixins.py

```diff
@@ -288,6 +289,15 @@ def __init__(self, address, name, shared_access_key_name=None,
             'key_name': shared_access_key_name,
             'shared_access_key': shared_access_key_value}
 
+        transport_type = kwargs.get('transport_type') or None
+        if transport_type is not None:
+            if transport_type.lower() == "amqpoverwebsocket":
+                self.auth_config['transport_type'] = TransportType.AmqpOverWebsocket
```

Comment:
I'd like to see `TransportType` being exposed at module level -- put it into `__init__.py` under azure.servicebus:
```
from uamqp.constants import TransportType
...
__all__ = [
   ...
   "TransportType"
]
```

Such that users can access the type through Enum instead of string.

Reply:
Done.